### PR TITLE
add the to_address into this spell

### PR DESCRIPTION
**Pull request by lorenz234**
### Update!
Please build spells in the proper [subproject](../dbt_subprojects/) directory. For more information, please see the main [readme](../README.md), which also links to a GH discussion with the option to ask questions.

### Contribution type
Please check the type of contribution this pull request is for:

- [ ] New spell(s)
- [x ] Adding to existing spell lineage
- [ ] Bug fix

**Note:** You can safely discard any section below which doesn't apply based on selection above

---

### For new spell(s)
If you are building new spell(s), please provide the following information:

- **Spell name(s):** *schema.table_name*
- **Description:** [Detailed description of the new spell(s) and their purpose]
- **Who are the new spell(s) for?** [Internal team or general community]
- **How will the new spell(s) be used downstream?** [Description of downstream usage]
- **Implementation details:** [Information on how the spell(s) are implemented]
- **Test instructions:** [How to test the new spell(s)]
- **Related issue(s):** [Link to related issues, if any]

---

### For adding to existing spell lineage
If you are adding to an existing spell lineage, please provide the following information:

- **Description:** [Description of the changes made]

---

### For bug fixes
If you are fixing a bug, please provide the following information:

- **Description:** [Description of the bug fix]
- **Steps to reproduce:** [How to reproduce the bug]
- **Implementation details:** [Information on how the bug was fixed]
- **Test instructions:** [How to test the fix]
- **Related issue(s):** [Link to related issues, if any]

---

### Additional information
Please provide any additional information that might help us review your pull request:

- [Any additional information]

---

Thank you for your contribution!

### CODE REVIEW

**Comment by lorenz234** [comment]
I would like to include the 'to_address' in this SQL query. It was originally defined in the schema, but it seems we forgot to add it to the SQL statement.

File: dbt_subprojects/hourly_spellbook/models/_sector/blobs/ethereum/_schema.yml [hunk 1]
```diff
@@ -85,9 +85,7 @@ models:
         description: "hash of the L1 transaction"
       - *beacon_slot_number
       - *beacon_epoch
-      - name: tx_from
-        description: "address of the sender"
-      - name: tx_to
```
Comment (Hosuke) [inline dbt_subprojects/hourly_spellbook/models/_sector/blobs/ethereum/_schema.yml:90]: Removal due to neither `tx_from` nor `tx_to` being present in the final selection.

**Review by Hosuke — APPROVED** [review]
LGTM.✅
Thank you @lorenz234

**Comment by jeff-dude** [comment]
@lorenz234 looks like there are downstream models of this one:
```
ethereum_blobs
rollup_economics_ethereum_l1_blob_fees
rollup_economics_ethereum_l1_fees
```

plz validate none of these need this column pulled through

**Review by jeff-dude — APPROVED** [review]

**Pull request merged**